apps/nowcasting-app/pages/index.tsx

```diff
@@ -112,11 +114,15 @@ export default function Home({ dashboardModeServer }: { dashboardModeServer: str
     setArraySettingInCookieStorage(CookieStorageKeys.VISIBLE_LINES, visibleLines);
   }, [visibleLines]);
 
-  // On view change, unset the clicked "GSP" if the aggregation is not GSP
+  // On view change, unset the clicked GSP if the aggregation is not GSP,
```

Comment:
should this comment be `unset the clicked region...` rather than `unset the clicked GSP ...`